resources/session02/homework/http_server.py

```diff
@@ -35,7 +40,19 @@ def parse_request(request):
 
 def resolve_uri(uri):
     """This method should return appropriate content and a mime type"""
-    return b"still broken", b"text/plain"
+    home = os.getcwd() + "/webroot"
+    path = home + uri
```

Comment:
What if uri is a relative path (something like `images/image.jpeg` instead of `/images/image.jpeg`)?